addons/website_crm/tests/test_website_crm.py

```diff
@@ -8,30 +8,11 @@
 class TestWebsiteCrm(odoo.tests.HttpCase):
 
     def test_tour(self):
-        self.start_tour("/", 'website_crm_tour')
+        self.start_tour("/", 'website_crm_tour', login='admin')
 
         # check result
         record = self.env['crm.lead'].search([('description', '=', '### TOUR DATA ###')])
         self.assertEqual(len(record), 1)
         self.assertEqual(record.contact_name, 'John Smith')
         self.assertEqual(record.email_from, 'john@smith.com')
         self.assertEqual(record.partner_name, 'Odoo S.A.')
-
-    def test_catch_logged_partner_info_tour(self):
-        user_login = 'admin'
-        user_partner = self.env['res.users'].search([('login', '=', user_login)]).partner_id
-        partner_email = user_partner.email
-        partner_phone = user_partner.phone
-
-        # no edit on prefilled data from logged partner : propagate partner_id on created lead
-        self.start_tour("/", "website_crm_catch_logged_partner_info_tour", login=user_login)
-        created_lead = self.env['crm.lead'].search([('description', '=', '### TOUR DATA PREFILL ###')])
-        self.assertEqual(created_lead.partner_id, user_partner)
-
-        # edited contact us partner info : do not propagate partner_id on lead
-        self.start_tour("/", "website_crm_tour", login=user_login)
-        created_lead = self.env['crm.lead'].search([('description', '=', '### TOUR DATA ###')])
-        self.assertFalse(created_lead.partner_id)
-        # check partner has not been changed
-        self.assertEqual(user_partner.email, partner_email)
-        self.assertEqual(user_partner.phone, partner_phone)
```

Comment:
Can you explain to me what was this test about to be sure we have to remove it? :)

Reply:
website_crm added some required data (from connected user) to the contact form. This test was here to ensure we could create an opportunity only by filling the subject and body (because the other data were retrieved from the user log). Since it heavily depends on dynamic data, it can't work anymore and thus must be removed.